pkgs/tools/compression/bzip2/default.nix

```
@@ -44,6 +44,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postInstall = ''
+    ln -s libbz2.so.1.0.6 $out/lib/libbz2.so.1.0
```

This way we don't need to update the `postInstall` phase on every soname bump.

```suggestion
    ln -s libbz2.so.1.0.* $out/lib/libbz2.so.1.0
```

---

No, wildcards don't work this way.  Now there's no symlink at all.